src/utils/LogSplitter.ts

```diff
@@ -278,6 +334,13 @@ export function logSplitter(fightData: LogLine[], progressCallback?: (progress:
                     currentRaid = null;
                 }
             }
+            if (currentWaves) {
+                // assume success if region changes, although if someone teleports out of the waves...
```

Comment:
If you're using this to check for completion of the inferno you could check  `141684 08-18-2024 22:37:49	Duration: <col=ff0000>45:33.00</col> (new personal best)` or just the Zuk death log (though having the duration of the run seems useful to display in the UI). Then you could just assume if someone leaves the region early it's incomplete.